pom.xml

```diff
@@ -96,7 +96,7 @@
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>google-cloud-bigquery</artifactId>
-        <version>1.127.9</version>
+        <version>1.127.10</version>
```

Comment:
Why we got such diff?

Reply:
I don't know how that happened. I assumed it was because I the origin got changed. I will undo this change manually and see if it goes well